react/ConditionLayout.tsx

```diff
@@ -18,6 +19,13 @@ const ConditionLayout: FC<Props> = ({ children, subjects, values }) => {
     values,
   })
 
+  useEffectSkipFirstRender(() => {
```

Comment:
Why skip the first render?

Reply:
I should add a comment explaining this. It's just to prevent updating the `values` in the first render two times: in the `useReducer` initial value and in the `useEffect` for whenever `values` change.